pkg/engine/params_k8s.go

```diff
@@ -10,14 +10,26 @@ import (
 	"strings"
 	"time"
 
-	"github.com/Azure/go-autorest/autorest/to"
-
 	"github.com/Azure/aks-engine/pkg/api"
 	"github.com/Azure/aks-engine/pkg/api/common"
+	"github.com/Azure/go-autorest/autorest/to"
+	"github.com/pkg/errors"
 )
 
+var compToParamImage = map[string]string{
```

Comment:
Tiny thing. I have the impression that `componentToParamImage` may be slightly easier to read. 
In my head, `comp` did not immediately trigger `component` and I think that `component` was favored across the code base.

Reply:
I agree with this as well